src/hooks/viewport/useMediaQuery.tsx

```diff
@@ -1,11 +1,13 @@
 import { useCallback, useEffect, useState } from "react";
 
 type MediaType = "isMobile" | "isTablet" | "isLaptop";
+
 const mediaBreakpoints: Record<MediaType, string> = {
   isMobile: "(max-width: 320px)",
   isTablet: "(max-width: 768px)",
   isLaptop: "(max-width: 1024px)",
 };
```

Comment:
[NOTE] Maybe this `isMobile` width should be made as `$xs` in `src/scss/attendee/breakpoint.scss`

Reply:
added [a58b99b](https://github.com/sparkletown/sparkle/pull/2905/commits/a58b99bf83fb6ed1a9b39ec673fb2882dab9e8e9)